python/ray/util/sgd/v2/backends/backend.py

```diff
@@ -88,6 +88,7 @@ def initialize_session(world_rank, train_func, checkpoint):
                     training_func=train_func,
                     world_rank=world_rank,
                     checkpoint=checkpoint,
+                    convert_checkpoint_func=self._backend.convert_checkpoint,
                     detailed_autofilled_metrics=use_detailed_autofilled_metrics
                 )
```

Comment:
Do we need to `convert` the loaded checkpoint on the driver and `unconvert` on the worker session?